rust/agama-server/src/network/model.rs

```diff
@@ -1259,3 +1302,11 @@ impl fmt::Display for InfinibandTransportMode {
         write!(f, "{}", name)
     }
 }
+
+#[derive(Debug, Clone, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub enum NetworkChange {
```

Comment:
The separate name in `DeviceUpdated` is here because it might happen that the device gets renamed, so the name included in the object is the new one and we cannot do the matching.

Reply:
OK, so these signatures are the minimal needed ones and it is not quite obvious

Reply:
I will add a comment to make it clear. We could even have a separate variant for the renaming, but I would rather prefer to keep it as it is (unless there is a good reason to change it).